docs/api/process.md

```diff
@@ -62,6 +62,14 @@ for otherwise it is `undefined`.
 When app is started by being passed as parameter to the default app, this
 property is `true` in the main process, otherwise it is `undefined`.
 
+### `process.platform`
+
+Returns the name of the platform the app is running on. `darwin` for macOS, `linux` for Linux and `win32` for Windows.
+
+### `process.argv`
+
+An array containing the arguments passed to the application when started. To get the arguments from the renderer process use `require('electron').remote.process.argv`
```

Comment:
In order for the docs linter to correctly understand these properties this needs to be

```markdown
A String[] array containing...
```